_python_workout_code/6.1.py

```diff
@@ -0,0 +1,21 @@
+def  myxml(*args, **kwargs):
+    result = ''
+    tag = getString(args, 0)
+    text = getString(args, 1)
+    attributes = ''
+    for i in kwargs:
+        attributes += ' ' + i + '="' + str(kwargs[i]) + '"'
```

Comment:
take a look at the f-strings: https://realpython.com/python-f-strings/ . It's more modern a convenient way to format strings in Python